mkdocs.yml

```diff
@@ -14,6 +14,8 @@ google_analytics:
 markdown_extensions:
   - codehilite
   - admonition
+  - pymdownx.superfences
```

Comment:
what is this please?

Reply:
It seems as if this is needed so that admonitions can render code blocks. (The ones enclosed in triple backticks, formally referred to as a "code fence")